pkg/scheduler/framework/plugins/volumebinding/volume_binding_test.go

```diff
@@ -138,6 +138,22 @@ func TestVolumeBinding(t *testing.T) {
 				podVolumesByNode: map[string]*scheduling.PodVolumes{},
 			},
 		},
+		{
+			name:                "PVC does not exist",
+			pod:                 makePod("pod-a", []string{"pvc-a"}),
+			node:                &v1.Node{},
+			pvcs:                []*v1.PersistentVolumeClaim{},
+			wantPreFilterStatus: framework.NewStatus(framework.UnschedulableAndUnresolvable, `persistentvolumeclaim "pvc-a" not found`),
+		},
+		{
+			name: "Part of PVCs do not exist",
+			pod:  makePod("pod-a", []string{"pvc-a", "pvc-b"}),
+			node: &v1.Node{},
+			pvcs: []*v1.PersistentVolumeClaim{
+				makePVC("pvc-a", "pv-a", waitSC.Name),
+			},
+			wantPreFilterStatus: framework.NewStatus(framework.UnschedulableAndUnresolvable, `persistentvolumeclaim "pvc-b" not found`),
+		},
```

Comment:
when PVCs referenced by the pod do not exist, the plugin returns `Unschedulable` status. these two tests verify it.